vividus-tests/src/main/resources/story/integration/ActionSteps.story

```diff
@@ -0,0 +1,12 @@
+Description: Integration tests for ActionSteps class.
+
+Meta:
+    @epic vividus-plugin-web-app
+
+Scenario: Action verification MOVE_TO
+Given I am on a page with the URL 'https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_onmousemove_over_enter'
```

Comment:
what about using vividus-test-site?
3-rd party websites can be changed/updated unexpectedly, that introduces flakiness to our tests

Reply:
discussed privately